invenio_communities/records/records/systemfields/communities/manager.py

```diff
@@ -166,7 +166,7 @@ def default(self, community_or_id):
         not, then use ``.add(community, default=True)`` instead.
         """
         id_ = self._to_id(community_or_id)
-        if id_ not in self._communities_ids:
+        if id_ is not None and id_ not in self._communities_ids:
```

Comment:
Question: How can id_ be None? if that happens I would say it's a mistake when calling this setter. In any case, if we want to control this error I would be in favour of raising another exception for this case something like 
```python
raise AttributeError(
                "Cannot set community as the default. "
                "Missing community_id."
            )
```

Reply:
We need to add possibility to remove the default community at all. In that case it will be None

Reply:
Ah right, I missunderstood it, it makes sense now, thanks :+1: